LogicalTypes.md

```diff
@@ -609,9 +609,20 @@ that is neither contained by a `LIST`- or `MAP`-annotated group nor annotated
 by `LIST` or `MAP` should be interpreted as a required list of required
 elements where the element type is the type of the field.
 
-Implementations should use either `LIST` and `MAP` annotations _or_ unannotated
-repeated fields, but not both. When using the annotations, no unannotated
-repeated types are allowed.
+```
+// List<Integer> (non-null list, non-null elements)
+repeated int32 num;
+
+// List<Tuple<Integer, String>> (non-null list, non-null elements)
+repeated group my_list {
+  required int32 num;
+  optional binary str (STRING);
+}
+```
+
+For all fields in the schema, implementations should use either `LIST` and
```

Comment:
Explicitly state that this applies to `all fields in the schema`.

Reply:
👍I think this is neccessary, hope implemetation would follow this rule...

Reply:
-0 on this change. I don't think this is more clear and I would prefer not to have the churn.

Reply:
I think the original sentence is confusing and resulted in a relevant discussion: https://github.com/apache/parquet-format/pull/466#pullrequestreview-2432448521. Therefore I prefer to keep the current change.